### ISSUE TITLE
FNX3-16851 ⁃ [Bug] Crash when taping on Settings -> Help

### ISSUE THREAD

**Issue by isabelrios** [mozilla-mobile/fenix#3202]
## Steps to reproduce
Go to Settings -> Help

### Expected behavior
The menu is open

### Actual behavior
The app crashes

### Device information
* Android device: Pixel 3, Android 9
* Fenix version: 6/3 build and also on latest 6/6 build

**Comment by isabelrios** [comment]
Logs: 
```
....
-06 11:02:51.085   820  3156 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_MODULE_CONNECTION_SIZE
06-06 11:02:51.085   820  3156 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_MODULE_CONNECTION
06-06 11:02:51.085   820  3156 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_CONNECTIONS_SIZE
06-06 11:02:51.085   820  3156 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPHS_CONNECTION
06-06 11:02:51.086   820  3156 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_CONNECTIONS_SIZE
06-06 11:02:51.086   820  3156 E         : __gcs_get_sg_conn_and_update_pld: subgraph conn size 0. Exiting
06-06 11:02:51.086   820  3156 D         : pktzr_send_cmd: opcode = 0x3, size = 452, token = 92
06-06 11:02:51.099   820  1184 D         : ipc_resp_thread: opcode: 0x2, payload_size: 4, resp_buf size: 16
06-06 11:02:51.099   820  1184 D         : pktzr_resp_cb: opcode = 0x2 size = 4 token = 92
06-06 11:02:51.099   820  1184 D         : pktzr_resp_cb: token 92 matched
06-06 11:02:51.099   820  3156 D sound_trigger_hw: reg_hal_event_session: ST_EVENT_SESSION_REGISTER capture_handle 62 p_ses 0xec774a00
06-06 11:02:51.099   820  3156 D sound_trigger_hw: session[1]: loaded_state_fn ---> active_state_fn
06-06 11:02:51.099   820  3156 D sound_trigger_hw: stdev_start_recognition:[1] Exit
06-06 11:02:51.100  1261  6612 I SoundTriggerHelper: startRecognition successful.
06-06 11:02:51.100  2477  2477 I GsaVoiceInteractionSrv: Start Hotword Recognition Status: true
06-06 11:02:51.107  2281  3493 I PlaceInferenceEngine: [anon] Changed inference mode: 1
06-06 11:02:51.108  2281  3493 I PlaceInferenceEngine: [account#-29786098] Changed inference mode: 1
06-06 11:02:51.109 16901 16901 I AppServices: Using viaduct_lib_name: fenix
06-06 11:02:51.112  2281  3493 I PlaceInferenceEngine: [account#135875463] Changed inference mode: 1
06-06 11:02:51.130  1261  1284 W ActivityManager: setHasOverlayUi called on unknown pid: 11405
06-06 11:02:51.130   624   646 W SurfaceFlinger: Attempting to set client state on removed layer: org.mozilla.fenix/org.mozilla.fenix.HomeActivity#0
06-06 11:02:51.130   624   646 W SurfaceFlinger: Attempting to destroy on removed layer: org.mozilla.fenix/org.mozilla.fenix.HomeActivity#0
06-06 11:02:51.132   624  2202 W SurfaceFlinger: Attempting to destroy on removed layer: AppWindowToken{f1775a6 token=Token{2ae7301 ActivityRecord{895f0e8 u0 org.mozilla.fenix/.HomeActivity t1122}}}#0
06-06 11:02:51.132   624  2202 W SurfaceFlinger: Attempting to destroy on removed layer: Task=1122#0
06-06 11:02:51.150 16901 16918 D NetworkSecurityConfig: No Network Security Config specified, using platform default
06-06 11:02:51.160 16901 16901 I AppServices: Using rc_log_ffi_lib_name: fenix
06-06 11:02:51.162 16901 16901 I rc_log_ffi::ios: rc_log adapter initialized!
06-06 11:02:51.174 16901 16901 W System.err: SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
06-06 11:02:51.174 16901 16901 W System.err: SLF4J: Defaulting to no-operation (NOP) logger implementation
06-06 11:02:51.174 16901 16901 W System.err: SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
06-06 11:02:51.174 16901 16901 D io.sentry.android.AndroidSentryClientFactory: Construction of Android Sentry.
06-06 11:02:51.175 16901 16901 D io.sentry.android.AndroidSentryClientFactory: Sentry init with ctx='org.mozilla.fenix.FenixApplication@1bf05b8' and dsn='Dsn{uri=https://sentry.prod.mozaws.net/}'
06-06 11:02:51.184 16901 16901 D io.sentry.android.AndroidSentryClientFactory: Using buffer dir: /data/user/0/org.mozilla.fenix/cache/sentry-buffered-events
06-06 11:02:51.191 16901 16901 W sh.CrashHandler: type=1400 audit(0.0:516): avc: denied { read } for name="version" dev="proc" ino=4026532129 scontext=u:r:untrusted_app:s0:c171,c256,c512,c768 tcontext=u:object_r:proc_version:s0 tclass=file permissive=0
06-06 11:02:51.201  2281  3493 I Places  : ?: PlacesBleScanner start() with priority 2
06-06 11:02:51.202  2281  3493 I PlaceInferenceEngine: [anon] Changed inference mode: 1
06-06 11:02:51.202  2281  3493 I PlaceInferenceEngine: [account#-29786098] Changed inference mode: 1
06-06 11:02:51.202  2281  3493 I PlaceInferenceEngine: [account#135875463] Changed inference mode: 1
06-06 11:02:51.207 16901 16929 E mozac/CrashReporter: CrashHandlerService received native code crash
06-06 11:02:51.208  2281  3493 I Places  : Converted 0 out of 4 WiFi scans
06-06 11:02:51.208 16901 16929 I mozac/CrashReporter: Received crash: NativeCodeCrash(minidumpPath=/data/user/0/org.mozilla.fenix/files/mozilla/vukrv4eo.default/minidumps/16b4721a-13d8-32e7-bb04-fd4065530e47.dmp, minidumpSuccess=true, extrasPath=/data/user/0/org.mozilla.fenix/files/mozilla/vukrv4eo.default/minidumps/16b4721a-13d8-32e7-bb04-fd4065530e47.extra, isFatal=true)
06-06 11:02:51.208 16901 16929 I mozac/CrashReporter: Showing prompt
06-06 11:02:51.209  1261  4757 I ActivityManager: START u0 {flg=0x10018000 cmp=org.mozilla.fenix/mozilla.components.lib.crash.prompt.CrashReporterActivity (has extras)} from uid 10171
06-06 11:02:51.215   837   837 D android.hardware.power@1.3-service.crosshatch-libperfmgr: LAUNCH ON
06-06 11:02:51.223 11271 11271 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@e602a19
06-06 11:02:51.225 16901 16929 I sh.CrashHandle: System.exit called, status: 0
06-06 11:02:51.225 16901 16929 I AndroidRuntime: VM exiting with result code 0, cleanup skipped.
06-06 11:02:51.228  2281 12218 I PlaceInferenceEngine: No beacon scan available - ignoring candidates.
06-06 11:02:51.240  1261  2519 I ActivityManager: Process org.mozilla.fenix:mozilla.components.lib.crash.CrashHandler (pid 16901) has died: prcp FGS
06-06 11:02:51.240  1261  2519 W ActivityManager: Scheduling restart of crashed service org.mozilla.fenix/mozilla.components.lib.crash.handler.CrashHandlerService in 1000ms
06-06 11:02:51.240  1261  2519 W ActivityManager: Bringing down service while still waiting for start foreground: ServiceRecord{791e1c6 u0 org.mozilla.fenix/mozilla.components.lib.crash.handler.CrashHandlerService}
06-06 11:02:51.240  1261  1279 W libprocessgroup: kill(-16901, 9) failed: No such process
06-06 11:02:51.241   809   809 I Zygote  : Process 16901 exited cleanly (0)
06-06 11:02:51.245  2281  2281 I BeaconBle: Client requested scan, settings=BleSettings [scanMode=ZERO_POWER, callbackType=ALL_MATCHES, reportDelayMillis=0, 1 filters, 0 clients, callingClientName=Places]
06-06 11:02:51.261  1261  1279 W libprocessgroup: kill(-16901, 9) failed: No such process
06-06 11:02:51.261  1261  1279 I libprocessgroup: Successfully killed process cgroup uid 10171 pid 16901 in 20ms
06-06 11:02:51.265  2281 12218 I PlaceInferenceEngine: No beacon scan available - ignoring candidates.
06-06 11:02:51.290  1261  1275 I ActivityManager: Displayed org.mozilla.fenix/mozilla.components.lib.crash.prompt.CrashReporterActivity: +75ms
06-06 11:02:51.296  2281 12218 I PlaceInferenceEngine: No beacon scan available - ignoring candidates.
06-06 11:02:51.315   837   837 D android.hardware.power@1.3-service.crosshatch-libperfmgr: LAUNCH OFF
06-06 11:02:51.331  2477  2477 I GsaVoiceInteractionSrv: Handling ACTION_STOP_DSP_HOTWORD
06-06 11:02:51.333   820  3156 D sound_trigger_hw: stdev_stop_recognition:[1] Enter
06-06 11:02:51.333   820  3156 D sound_trigger_hw: active_state_fn:[1] handle event id 7
06-06 11:02:51.333   820  3156 D sound_trigger_hw: active_state_fn:[1] unhandled event
06-06 11:02:51.333   820  3156 D sound_trigger_hw: active_state_fn:[1] handle event id 4
06-06 11:02:51.333   820  3156 D sound_trigger_hw: dereg_hal_event_session: ST_EVENT_SESSION_DEREGISTER capture_handle 62 p_ses 0xec774a00
06-06 11:02:51.333   820  3156 D sound_trigger_hw: stop:[1] gcs stop VA engine, size=24
06-06 11:02:51.333   820  3156 D         : pktzr_send_cmd: opcode = 0x5, size = 24, token = 93
06-06 11:02:51.347   820  1184 D         : ipc_resp_thread: opcode: 0x2, payload_size: 4, resp_buf size: 16
06-06 11:02:51.347   820  1184 D         : pktzr_resp_cb: opcode = 0x2 size = 4 token = 93
06-06 11:02:51.347   820  1184 D         : pktzr_resp_cb: token 93 matched
06-06 11:02:51.348   820  3156 D sound_trigger_hw: stop:[1] calling gcs_disable with handle 1
06-06 11:02:51.348   820  3156 E         : __gcs_disable_update_pld: sgt_array[i] = 0x1
06-06 11:02:51.348   820  3156 E         : __gcs_disable_update_pld: sgt_array[i] = 0x2
06-06 11:02:51.348   820  3156 D         : pktzr_send_cmd: opcode = 0x4, size = 80, token = 94
06-06 11:02:51.358   820  1184 D         : ipc_resp_thread: opcode: 0x2, payload_size: 4, resp_buf size: 16
06-06 11:02:51.359   820  1184 D         : pktzr_resp_cb: opcode = 0x2 size = 4 token = 94
06-06 11:02:51.359   820  1184 D         : pktzr_resp_cb: token 94 matched
06-06 11:02:51.359   820  3156 D sound_trigger_hw: set_device: disable device (1) = listen-cpe-handset-mic
06-06 11:02:51.360   820  3156 D sound_trigger_hw: session[1]: active_state_fn ---> loaded_state_fn
06-06 11:02:51.360   820  3156 D sound_trigger_hw: stdev_stop_recognition:[1] Exit status 0
06-06 11:02:51.362  2477  2477 I GsaVoiceInteractionSrv: Stop Hotword Recognition status: true
06-06 11:02:53.085  1261  1261 V SettingsProvider: Notifying for 0: content://settings/system/screen_brightness
06-06 11:02:53.367   838   931 I CHRE    : @ 2191.117: [AR_CHRE] still: 100
```

**Comment by colintheshots** [comment]
I cannot seem to reproduce this today with the 6/6 build on my Pixel 3 XL running Android 10. The stack trace seems to point at a native crash, likely on content, which is curious.

**Comment by vesta0** [comment]
@isabelrios are you able to reproduce this with today's build?

**Comment by isabelrios** [comment]
Hi @vesta0 , yes, I can reproduce it with today's build. I forgot to mention I have the device in Spanish, in case it makes a difference. @sv-ohorvath was able to repro too, I talked to her before filing the issue. Let me know if you need more logs or info.

**Comment by vesta0** [comment]
@colintheshots please let @isabelrios know if more logs can be provided to help investigate this. 

FYI I just tried again and couldn't reproduce this on Android 9, Pixel 3, Spanish settings.

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest Nightly **Build #11970610 from 16/7** using the following devices:
• Google Pixel 3a (Android 9)
• Google Pixel 3XL (Android 9)
• Google Pixel 3 (Android 9)

From the language setup point of view I've selected  Español (España)
![rsz_screenshot_20190716-1559311111](https://user-images.githubusercontent.com/51314259/61297779-9cb57d80-a7e5-11e9-8da2-20c7569742e5.png)

► Video from Google Pixel 3a 
![20190716_160252](https://user-images.githubusercontent.com/51314259/61297849-c8d0fe80-a7e5-11e9-84ee-5b8137f8ebd6.gif)
